code/script.js

```diff
@@ -109,36 +109,27 @@ function changeHeaderInnerHTML(data, timeOfSunrise, timeOfSunset) {
 };
 
 //Changes background image in header depending on todays weather prognosis
-function setBackgroundWeather(data) {
+function getBackgroundWeatherClass(data) {
     let main = data.weather.map((item) => item.main);
     if (main.includes("Clouds")) {
-        weatherImg = cloudyIcon;
-        backgroundImg.classList = "clouds";
+        return "clouds";
```

Comment:
these now return instead of assigning, the assigning is done when we call the function